WcaOnRails/spec/models/competition_wcif_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+ONE_MINUTE_IN_CENTISECONDS = 60*100
```

Comment:
I feel weird too, but I can make myself believe it's OK if the term inherently must have lower case. Changing `milli` to `mega` should override that convention! Not that I can remember ever seeing it used.

I don't know what classes are involved. Whatever makes `expect(7.minutes.in_cs).to eq(42000)` go green!